server/src/core/server/graph/resolvers/PremoderateEmailAddress.ts

```diff
@@ -0,0 +1,12 @@
+import {
+  GQLPremoderateEmailAddressConfig,
+  GQLPremoderateEmailAddressConfigTypeResolver,
+} from "coral-server/graph/schema/__generated__/types";
+
+export const SlackConfiguration: GQLPremoderateEmailAddressConfigTypeResolver<GQLPremoderateEmailAddressConfig> =
```

Comment:
looks like this should be updated to something like `PremoderateEmailAddress` instead of `SlackConfiguration`? :)

Reply:
renamed!